test/unit/dimensions.js

```diff
@@ -461,7 +462,9 @@ test( "allow modification of coordinates argument (gh-1848)", 1, function() {
 		return coords;
 	});
 
-	equal( element.offset().top, 100, "coordinates are modified" );
+	offsetTop = element.offset().top;
+	ok( Math.abs(offsetTop - 100) < 0.02,
```

Comment:
In IE11 it was `~0.011` so I just rounded. :P I'll happily take a more substantiated number.